definitions/procedures/pulpcore/rpm_datarepair.rb

```diff
@@ -0,0 +1,22 @@
+module Procedures::Pulpcore
+  class RpmDatarepair < ForemanMaintain::Procedure
+    include ForemanMaintain::Concerns::SystemService
+    include ForemanMaintain::Concerns::PulpCommon
+
+    metadata do
+      description 'Run pulpcore-manager rpm-datarepair 4073'
+      for_feature :pulpcore
+    end
+
+    def run
+      with_spinner('Running pulpcore-manager rpm-datarepair 4073') do |spinner|
+        necessary_services = feature(:pulpcore_database).services
+
+        feature(:service).handle_services(spinner, 'start', :only => necessary_services)
```

Comment:
Is this necessary if this is set to execute after `Procedures::Service::Start`, assuming the steps are processed sequentially?